decidim-admin/lib/decidim/admin/test/manage_moderations_examples.rb

```diff
@@ -1,5 +1,38 @@
 # frozen_string_literal: true
 
+shared_examples "sorted moderations" do
+  let!(:moderations) do
+    reportables.first(reportables.length - 1).map do |reportable|
+      moderation = create(:moderation, reportable: reportable, report_count: 1, reported_content: reportable.reported_searchable_content_text)
+      create(:report, moderation: moderation)
+      moderation
+    end
+  end
+  let!(:moderation) { moderations.first }
+  let!(:hidden_moderations) do
```

Comment:
Is there any reason that I'm missing why we're adding the '`hidden_moderations`' but not using it?